### ISSUE TITLE
How to use with server hosting???

### ISSUE THREAD

**Issue by MsDemonicRaven** [webbukkit/dynmap#2292]
https://github.com/webbukkit/dynmap/wiki/Dynmap-On-Xenon-Hosting-Service%2C--courtesy-of-mavbear

I read the whole wiki on how to do it...but the first download link doesnt work. it comes up as 403 forbidden. im really....really confused by the whole article. 

can someone dumb it down for me? I just need someone to do that so when the stable version of this comes out, i know what to do. Im using mcprohosting if that helps.

**Comment by TomSputz** [comment]
This should be able to help - [MCProhosting's tutorial](https://clients.mcprohosting.com/knowledgebase/43/How-to-Set-Up-DynMap.html), if you still need help, add me on discord with MoreThanTom#0828, and I'll help however I can

**Comment by MsDemonicRaven** [comment]
Im adding u right now. I cant find the configuration.txt file

**Comment by generrosity** [comment]
Resolved? Any tips on what is needed on the wiki page? I can update it but I don't know the details.
